Providers/Modules/Plugins/Common/plugin/out_oms_blob.rb

```diff
@@ -85,6 +85,9 @@ def create_blob_put_request(uri, msg, request_id, file_path = nil)
       headers["Content-Type"] = "application/octet-stream"
       headers["Content-Length"] = msg.bytesize.to_s
 
+      # If the request version is 2011-08-18 or newer, the ETag value will be returned
+      headers[OMS::CaseSensitiveString.new("x-ms-version")] = "2011-08-18"
```

Comment:
can you please verify if this is a required change or an optional one. if it is required, we also need to modify out_oms_changetracking.rb as well. the change should be similar, since the code is very similar